pwnlib/util/sh_string.py

```diff
@@ -423,6 +436,8 @@ def sh_string(s):
     if quoted:
         quoted_string += SINGLE_QUOTE
 
+    if isinstance(orig_s, six.binary_type):
```

Comment:
I'd strongly recommend sticking to whatever convention is used by `six`, if only because it's well-supported and documented.

Reply:
It is actually fully equivalent:
```py
>>> six.binary_type is bytes # python 3
True
>>> six.binary_type is bytes # python 2
True
```
so I think that using `bytes` is preferable (it could be up to 2x faster if this is important).